File: app/src/main/java/com/karim/booksapp/data/models/pojos/Epub.java

package com.karim.booksapp.data.models.pojos;

import com.squareup.moshi.Json;

public class Epub {

    @Json(name = "isAvailable")
    public Boolean isAvailable;
    @Json(name = "acsTokenLink")
    public String acsTokenLink;

}

File: app/src/main/java/com/karim/booksapp/data/models/pojos/Epub.kt
package com.karim.booksapp.data.models.pojos

import com.squareup.moshi.Json

class Epub {

    @Json(name = "isAvailable")
    var isAvailable: Boolean? = null
    @Json(name = "acsTokenLink")
    var acsTokenLink: String? = null

}
